src/css/_sn.scss

```diff
@@ -198,3 +198,49 @@
   @extend .border-0;
   @extend .slide-down-animation;
 }
+
+/**
+ * A button that is just an icon. Separated from .sn-button because there
+ * is almost no style overlap.
+ */
+.sn-icon-button {
+  color: var(--sn-stylekit-grey-1);
```

Comment:
We should a utility class for this color and extend it here instead, so that the color class itself can be reused.

Reply:
Yeah, I left it like this because I couldn't find a theme consistent name for it. Any ideas?

Reply:
Just color-grey-1 would be okay I think. We'll probably have to rethink the whole color scheme once all our colors are v4

Reply:
Done in amended commit.